### PR TITLE
test the welcome screen

### PR DESCRIPTION
**Pull request by ziyadbagh**
The implemented test provides 91% line coverage for the WelcomeScreen by verifying both visual elements (the animated image and the welcome text) are correctly rendered when the screen loads.

Additionally, for clarity and consistency, the test file originally named LoginTest has been renamed to LoginScreenTest to better reflect its content and purpose. This change improves code maintainability and readability.

### CODE REVIEW

**Comment by ziyadbagh** [comment]
> Can you create an issue and link it to this PR and the related task? Remeber to capitalize your commits.

done

File: app/src/androidTest/java/com/github/se/signify/ui/screens/LoginScreenTest.kt
Comment (ThomasMPont) [inline app/src/androidTest/java/com/github/se/signify/ui/screens/LoginScreenTest.kt:1]: Changing older tests for consistency is good practice, but you should have done the change in a separate commit.
Aim for more smaller commits a PR.

File: app/src/androidTest/java/com/github/se/signify/ui/screens/WelcomeScreenTest.kt
Comment (ThomasMPont) [inline app/src/androidTest/java/com/github/se/signify/ui/screens/WelcomeScreenTest.kt:1]: Splitting your test into multiple tests that only check one element of the sceen might make for easier debugging for future developers. I'll let it slide since the test is very short.
To better organize your own work, you could have split the making of this file into three commits.
- Create welcome screen test
- Test welcome screen image
- Test welcome screen text

**Review by ThomasMPont — APPROVED** [review]
The new test looks good.
I think you could have done better with commits though.
I'm approving this as the additions are small, but you could split the test into two smaller tests, as I suggested, before merging if you want.
